framework/content/ecl-typography/ecl-typography-headings/_headings.scss

```diff
@@ -1,46 +1,75 @@
 // Headers
-.ecl-h1,
-.ecl-h2,
-.ecl-h3,
-.ecl-h4,
-.ecl-h5,
-.ecl-h6 {
+.ecl-h {
```

Comment:
`h` means `heading` or `height` or `header`? IMHO we should make this class name more explicit